cmd/kube-controller-manager/app/options/options.go

```diff
@@ -249,6 +252,7 @@ func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledBy
 	s.SAController.AddFlags(fss.FlagSet("serviceaccount controller"))
 	s.TTLAfterFinishedController.AddFlags(fss.FlagSet("ttl-after-finished controller"))
 	s.Metrics.AddFlags(fss.FlagSet("metrics"))
+	s.Logs.AddFlags(fss.FlagSet("logs"))
```

Comment:
Does this fit better into generic? Is it just one flag? Where are the other klog flags?

Reply:
Othe klog flags are Global flags. I think this flag should be in same flagSet `logs`  as in other control plane components.

Reply:
https://gist.github.com/fd93b5d6044d7c256d2e70fcabf503ae does it. All logging flags will be under the logs section.

Reply:
My gist does not move `log-flush-frequency` yet. That's also defined in component-base/logs and has to go to the named flag set.

Reply:
As the proposed change is bigger (removes the init func behaviour of the package), I am fine to postpone this to 1.20. But would like to see a PR started with the gist, tagged with the relevant people **before** approving this PR here.

Reply:
Done https://github.com/kubernetes/kubernetes/pull/92707

Reply:
We agreed to keep this new logs section for now, and move klog flags over in a follow-up (in 1.19 or 1.20 if we miss code freeze).